src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigrateScriptsRule.cs

```diff
@@ -94,6 +99,16 @@ private bool CommandIsNotNeededInMSBuild(string command)
             return command.Contains("dotnet publish-iis");
         }
 
+        private static bool IsRazorPrecompilationCommand(string command)
+        {
+            return command.Contains("dotnet razor-precompile");
+        }
+
+        private static void EnableRazorCompilationOnPublish(ProjectRootElement csproj)
+        {
+            csproj.AddProperty("MvcRazorCompileOnPublish", "true");
```

Comment:
We generally care about it looking pretty. Is there a more correct property group than the `first` one?